graylog2-web-interface/src/components/navigation/ThemeModeToggle.jsx

```diff
@@ -2,7 +2,7 @@
 import * as React from 'react';
 import { useEffect, useState } from 'react';
 import styled, { css, withTheme, type StyledComponent } from 'styled-components';
-import debounce from 'lodash/debounce';
+import { defer } from 'lodash';
```

Comment:
To enable tree-shaking in lodash, you have to import functions using syntax like `import foo from 'lodash/foo'`. 
`import { foo } from 'lodash'` will not tree-shake.
```suggestion
import defer from 'lodash/defer';
```

Reply:
:white_check_mark: